src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java

```diff
@@ -91,9 +91,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
             temp.substring(8, 12),
             temp.substring(12, 16)
         ));
+        setDssMetaData(data);
 
         return AboutToStartOrSubmitResponse.<CaseData, State>builder()
             .data(data)
             .build();
     }
+
+    private void setDssMetaData(CaseData data) {
+
+        data.setDssQuestion1("Full Name");
+        data.setDssQuestion2("Date of Birth");
+        data.setDssQuestion3("Test");
+        data.setDssAnswer1("case_data.childrenFirstName, case_data.childrenLastName");
+        data.setDssAnswer2("case_data.childrenDateOfBirth");
+        data.setDssAnswer3("case_data.otherAdoptionAgencyAddress.Country");
```

Comment:
@sairamkantheti1 , is country an object or a string?

Reply:
@yogendraU 
Country is a string. Actually it is lower case country. I updated with same

Reply:
@yogendraU  Sure. I will make this change. Please update the same in comments of below user story including the exact question label
https://tools.hmcts.net/jira/browse/ADOP-1973